fem/pbilinearform.hpp

```diff
@@ -241,6 +246,15 @@ class ParMixedBilinearForm : public MixedBilinearForm
        @a A. */
    void ParallelAssemble(OperatorHandle &A);
 
+   virtual void FormRectangularSystemMatrix(const Array<int> &trial_tdof_list,
+                                            const Array<int> &test_tdof_list,
+                                            OperatorHandle &A);
+
+   virtual void FormRectangularLinearSystem(const Array<int> &trial_tdof_list,
+                                            const Array<int> &test_tdof_list, Vector &x,
+                                            Vector &b, OperatorHandle &A, Vector &X,
+                                            Vector &B);
```

Comment:
Documentation would be helpful here. Even just a reference to look at the serial versions of these.